app/main.py

```
@@ -136,6 +137,22 @@ async def lifespan(app: FastAPI):  # noqa: ARG001
+        try:
+            recovered_transfers = remote_transfer_service.recover_interrupted_transfers(db)
+            if recovered_transfers:
+                logger.warning(
+                    "Recovered %s interrupted remote transfer job(s)", recovered_transfers
+                )
+            else:
+                logger.info("No interrupted remote transfers found")
+        finally:
+            db.close()
+    except SQLAlchemyError as e:
+        logger.warning(f"Error during remote transfer recovery: {e}")
```

Ruff (0.15.11)

[warning] Logging statement uses f-string

(G004)